relationships/candidates/AWSELASTICACHEREDISCLUSTER.yml

```diff
@@ -0,0 +1,20 @@
+category: AWSELASTICACHEREDISCLUSTER
+lookups:
+  - entityTypes:
+      - domain: INFRA
+        type: AWSELASTICACHEREDISCLUSTER
+    tags:
+      matchingMode: ALL
+      predicates:
+        - tagKeys: ["aws.elasticache.endpointAddress"]
+          field: cloud.resource_endpoint_address
+        - tagKeys: ["aws.elasticache.endpointPort"]
+          field: cloud.resource_endpoint_port
+        - tagKeys: ["aws.elasticache.engine"]
```

Comment:
This field is not needed, since the `AWSELASTICACHEREDISCLUSTER` is constrained to Redis.